modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc

```diff
@@ -1,9 +1,10 @@
-== Annex II. XML Examples
+= Annex II. XML Examples
+include::partial$attributes.adoc[]
 
-=== AII.1. Code List Examples
+== AII.1. Code List Examples
 
 [[access-right-xml-example,access-right xml Example]]
-==== AII.1.. access-right
```

Comment:
The numbering A II.1.. for all code list examples is a bit confusing but we can live with it for the time being.
Is it a reason, why it has to be like this?

Reply:
The intention was to give an specific numbering for the Annex; in the end it was never finished.

Reply:
Numbering fixed.